Source/items.cpp

```diff
@@ -2999,11 +3003,21 @@ void SpawnItem(Monster &monster, Point position, bool sendmsg)
 	} else if (monster.isUnique() || dropsSpecialTreasure) {
 		// Unqiue monster is killed => use better item base (for example no gold)
 		idx = RndUItem(&monster);
-	} else if (Quests[Q_MUSHROOM]._qactive == QUEST_ACTIVE && Quests[Q_MUSHROOM]._qvar1 == QS_MUSHGIVEN) {
+	} else if (dropBrain && !gbIsMultiplayer) {
 		// Normal monster is killed => need to drop brain to progress the quest
-		idx = IDI_BRAIN;
 		Quests[Q_MUSHROOM]._qvar1 = QS_BRAINSPAWNED;
+		NetSendCmdQuest(true, Quests[Q_MUSHROOM]);
+		// brain replaces normal drop
+		idx = IDI_BRAIN;

```

Comment:
since this is only for single player do we need to call netsend?

Reply:
Strictly speaking no we don't need it.
I would still keep it. It doesn't harm and when you search if you send all quest updates over the wire you don't have to think about this spot.
We also have another one that is not strictly necessary, cause it's only used in singleplayer (when spawning the Fungal Tome).